packages/core/src/components/item/item.scss

```diff
@@ -96,3 +96,7 @@ ion-input.item {
   background: transparent;
   cursor: pointer;
 }
+
+.item > .icon {
+  font-size: 1.6em;
```

Comment:
This does not work with icons that have `[slot="end"]` because it is inside of the inner item:

<img width="378" alt="screen shot 2018-01-11 at 1 36 04 pm" src="https://user-images.githubusercontent.com/6577830/34840895-72eb3cba-f6d4-11e7-8bae-644c9d5f80df.png">

I think it needs to go in here for each mode file: https://github.com/ionic-team/ionic/blob/core/packages/core/src/components/item/item.ios.scss#L94-L97

Reply:
Here is how v3 styled them:

```
.item > ion-icon, .item-inner > ion-icon {
    min-height: 2.8rem;
    font-size: 2.8rem;
    line-height: 1;
}
```